andela/vidar-ah-backend
# #164270478 Complete user registration and Email verification

#### What does this PR do?
It configures Authors Haven to use PostgreSQL as DB Engine.

#### Description of Task to be completed?
Users should be able to create accounts on Authors Haven and receive an email notification containing a verification link.
#### How should this be manually tested?
- Check .env.sample for the updated .env variables.
- Run `npm run db-migrate` to run migrations.
- Run `npm run start` to start the application.
- Make a POST request to `/api/v1/user` with a JSON object containing the following fields: `email`, `username`, `name` and `password`. They're all required. If one is not sent, there'll be an error.
- If this request is successful, you should receive an email notification with a verification link to verify your account. 
- Click the link sent in the email, it's going to verify your account and return a success message.
- If the account has already been verified or not found, you should see an error.

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
#164270478
#### Screenshots (if appropriate)
#### Questions:
N/A
